src/test/java/httpserver/HttpResponderTest.java

```diff
@@ -48,12 +56,27 @@ public boolean isClosed() {
     private class SimpleHttpResponsePresenter implements Presentable {
         private String response;
 
-        public SimpleHttpResponsePresenter(String response) {
-            this.response = response;
+        @Override
+        public String present(HttpResponse response) {
+            return response.responseLine + System.lineSeparator();
         }
+    }
+
+    private class SimpleHttpParser implements Parseable {
 
         @Override
-        public String present() {
+        public String parse(String request) {
+            String[] splitRequest = request.split(" ");
+            return splitRequest[1];
```

Comment:
Do we need this logic here? Or can we make this double simpler? I worry that this double is very similar to `HttpParser`. If it is duplicating behavior, I wonder what the value of this double is.

Reply:
Will simplify